src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlTree.java

```diff
@@ -222,6 +225,20 @@ public HtmlTree add(List<? extends Content> list) {
         return this;
     }
 
+    /**
+     * Adds each of a list of items, using a map function to create the content for each item.
+     *
+     * @param list   the list
+	 * @param mapper the map function to generate the content for each item
+	 *
+     * @return this object
+     */
+    @Override
+    public <T> HtmlTree addAll(Collection<T> list, Function<T, Content> mapper) {
+        list.forEach(item -> add(mapper.apply(item)));
+        return this;
+    }
```

Comment:
Another observable change would be that the method could not be used in method chaining, because of the different return type.  But having the method applicable at least in a limited form in ContentBuilder would be nice, but that suggests leaving a covariant override here.

Reply:
Another reason to make a minor update ... the doc comment on the method in Content is incorrect ... the `@implSpec` is wrong.

I'll copy (not move) the implementation from `HtmlTree` to `Content`.